packages/cf-component-card/src/CardToolbarLink.js

```diff
@@ -1,43 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Link from 'cf-component-link';
+import { createComponent } from 'cf-style-container';
 
-class CardToolbarLink extends React.Component {
-  constructor(props) {
-    super(props);
-    this.handleClick = this.handleClick.bind(this);
-  }
-
-  handleClick(e) {
-    e.preventDefault();
-    this.props.onClick();
-  }
-
-  render() {
-    let className = 'cf-card__toolbar_link';
-
-    if (this.props.isActive) {
-      className += ' cf-card__toolbar_link--open';
-    }
-
-    return (
-      <Link
-        role="tab"
-        id={this.props.id}
-        className={className}
-        onClick={this.handleClick}
-      >
-        {this.props.children}
-      </Link>
-    );
-  }
-}
+const CardToolbarLink = ({ className, href, children, target }) => (
+  <a
+    className={`cf-link ${className}`}
```

Comment:
I think I'd like to have access to all the attributes of `<a />` or just pass in the `<a />` component myself.

Could we either spread `this.props` on this tag or change it to be just a decorative component and pass in the `<a />` ourselves?